src/main/java/Duke.java

```diff
@@ -1,10 +1,169 @@
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+
 public class Duke {
+    final static String HORIZON = "____________________________________________________________\n";
+    static List<Task> tasks = new ArrayList<>();
+    static String withFormat(String str) {
+        return HORIZON + str + '\n' + HORIZON;
+    }
+
+    public static boolean isNumeric(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch(NumberFormatException e){
```

Comment:
perhaps include a spacing before and after () per the coding standard? :)

Reply:
Fixed, thank you!